development/create-static-server.js

```diff
@@ -20,6 +21,12 @@ const createStaticServer = (options) => {
         public: path.resolve('./node_modules'),
       });
     }
+
+    // Handle test-dapp-multichain URLs by removing the prefix
+    if (request.url.startsWith('/test-dapp-multichain/')) {
```

Comment:
How come this isn't required for the regular test dapp?

Reply:
Still confused by this

Reply:
just verified that the `multichain-test-dapp` package's `build/index.html` has paths that include `/test-dapp-multichain`. This may have been done due to how the page is published to `https://metamask.github.io/test-dapp-multichain/latest/` instead of `https://metamask.github.io/test-dapp-multichain/`. I'm going to create a ticket to see if this is addressable later

Reply:
https://github.com/MetaMask/MetaMask-planning/issues/4145

Reply:
Makes sense 👍